dvc/project.py

```diff
@@ -0,0 +1,203 @@
+import os
```

Comment:
Should we extract all the core code into `core` module?
I'm talking about `project.py`, `stage.py`, `state.py`...

Reply:
For the record: we talked about it during the meeting and decided to leave it as is.